src/interfaces.ts

```diff
@@ -13,13 +13,28 @@ export interface IGitHubRepoLabels {
   }
 }
 
+export enum MergeStateStatus {
+  BEHIND,
+  BLOCKED,
+  CLEAN,
+  DIRTY,
+  DRAFT,
+  HAS_HOOKS,
+  UNKNOWN,
+  UNSTABLE
+}
+
+export enum MergeableState {
+  CONFLICTING,
+  MERGEABLE,
+  UNKNOWN
```

Comment:
These are numeric enums, and you are using them to compare against strings

https://www.typescriptlang.org/docs/handbook/enums.html

Ex 

![image](https://user-images.githubusercontent.com/16867443/112739490-b0b1fc00-8f42-11eb-94a2-1fb5cd4a8655.png)